src/rewards/airdrop.ts

```diff
@@ -144,6 +162,23 @@ export interface AirdropTarget {
   recipient: PublicKey
 }
 
+export interface AirdropCreateParams {
+  /// The expiration time for the airdrop
+  expire_at: BN
+
+  /// The stake pool that claimed rewards are deposited into.
+  stake_pool: PublicKey
+
+  /// A description for this airdrop
+  short_desc: string
+
+  /// The bump seed needed to generate the airdrop account address
+  vault_bump: number
+
+  /// Airdrop settings
+  flags: BN
+}
```

Comment:
Nice Nice Nice

You can use jsdoc
```suggestion
export interface AirdropCreateParams {
  /** The expiration time for the airdrop /
  expire_at: BN

  /** The stake pool that claimed rewards are deposited into. */
  stake_pool: PublicKey

  /** A description for this airdrop */
  short_desc: string

  /** The bump seed needed to generate the airdrop account address */
  vault_bump: number

  /** Airdrop settings */
  flags: BN
}
```

Reply:
AirdropCreatParams removed because it is not needed for current jet-gov integration